### DISCUSSION TITLE
제페토 템플릿으로 프로젝트 생성시에 에러가 떠요

### DISCUSSION THREAD

**Discussion by mumumine** [naverz/zepeto-studio-kor#1174] [5. Q&A]
제페토 월드 템플릿 import 에러입니다ㅠ 
처음 패키지를 import를 완료하고나서 유니티를 종료하고 다시 열어서 제페토 템플릿으로 프로젝트를 생성했더니 저런 문구가 떠요
![image](https://user-images.githubusercontent.com/111327538/184782230-9e5ce487-8989-4cef-9aac-6e66db4e2218.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터님 
계속해서 위와 같은 오류가 발생하고 있는지요? 
유니티 버전 2020.3.9인지도 확인 부탁 드립니다.

**Reply by mumumine** [reply]
네 버전은 맞는데..  계속 안되다가... 몇번의 시도끝에 또 갑자기 되네요 ..! 답변 감사합니다ㅎㅎ

**Reply by Zepeto-TechEvangelist** [reply]
다행입니다~ 윈도우 환경에서 처음 템플릿 설치시 오류가 나는 경우가 종종 발생하고 있어서 저희쪽에서도 known issue로 관리 중입니다.
감사합니다.
